src/datacatalogtordf/distribution.py

```diff
@@ -119,7 +119,7 @@ class Distribution:
     _access_service: DataService
     _download_URL: URI
     _byte_size: Decimal
-    _spatial_resolution: Decimal
+    _spatial_resolution_in_meters: Decimal
```

Comment:
Den skal være liste for datasett og enkel for distribusjoner?

Reply:
Etter som jeg ser så har den også 0..n for distribusjoner: https://data.norge.no/specification/dcat-ap-no/#Distribusjon-romlig-oppl%C3%B8sning

Reply:
Bra du oppdaget det. Jeg fikser!!